config/resolve_test.go

```diff
@@ -3,8 +3,14 @@ package config
 import (
 	"bytes"
 	"context"
+	"encoding/json"
+	"github.com/aws/aws-sdk-go-v2/feature/ec2/imds"
+	"github.com/google/go-cmp/cmp"
 	"io/ioutil"
 	"net/http"
+	"net/http/httptest"
+	"os"
+	"strconv"
 	"testing"
```

Comment:
Nit should we all be using `goimports` to sort our import statements?  this update groups the new non-stdlib imports into the stdlib group.